### DISCUSSION TITLE
Show enum values as hex

### DISCUSSION THREAD

**Discussion by graealex** [EWSoftware/SHFB#977] [Q&A] [answered]
Some of my enums only make sense if presented as hexadecimal values. I found the FlagsEnumValueFormat property, but it poses two problems:

1. Some enums don't make particularly sense in hex, and should just be decimal - is it possible to apply the EnumValueFormat.HexValue only to certain enums? Not a super big deal, but it'd be nice if it could be applied depending on use scenario.
2. Despite having set <Argument Key="FlagsEnumValueFormat" Value="HexValue" /> for the documentation project, the CHM as well as the website HTMLs still have all enums in decimal. I am using the VS2013 presentation style. What could be the problem?

**Comment by EWSoftware** [comment, marked as answer]
The value format only applies to enumerations with the Flags attribute since they're typically the ones where its makes sense.  Any without it are shown in decimal since they're just values, not flags.  On those with the attribute, it's all or nothing since there's no way to indicate where it should or shouldn't apply the option.

**Reply by graealex** [reply]
Then I'll put the [Flags] on enums that I need being displayed as hex. Thanks!
